### DISCUSSION TITLE
The sgp conductivity is zero obtained by Q3D， and An error occurs when calculating  smr concuctivity.

### DISCUSSION THREAD

**Discussion by gswylq** [anharmonic/d3q#5] [General]
Hello everyone,
I  install Q3D+themal and run the example of Si， and the calculated results of Si is normal.
When I calculate the conductivity of an new material Sc2O2S, the results are abnormal. The sgp conductivity is zero obtained by Q3D， and An error occurs when calculating  smr conductivity.

The results and input files add to attachment.

Please give some help.

Thanks.

Qiang Li

[resutls.zip](https://github.com/anharmonic/d3q/files/8387992/resutls.zip)

![cgp conductivity](https://user-images.githubusercontent.com/84161015/161024397-7acb0fa5-9ee0-4dea-b2b2-07a8413c268f.png)
![phonon_1](https://user-images.githubusercontent.com/84161015/161024406-e4a083c8-ce36-45ae-bc17-61ada5e1f9f4.png)
![smR](https://user-images.githubusercontent.com/84161015/161024411-7f09c549-4076-4a8e-a3e9-dfdfa86b86a0.png)

**Comment by paulatz** [comment]
Dear Qiang Li,
I do not have time to look in very much detail in your email, but there are a few things that could be causing problems:
you are using a 2x2x2 grid for d3q. In some cases, including yours from what i see, all the 3rd order force constants are zero because of symmetry in a 2x2x2 grid, you may try with a 3x3x3 or 4x4x4 (it should not be very expensive)

Point 1 is by far the biggest problem, anyway, there are a couple more things that you should eventually fix:
you are treating your material as a metal (i.e. you have set a smearing for occupations). This can cause problems in both phonons and d3q calculations. Try to use fixed occupations.

you are using a very small grid of 10x10x1 or 5x5x5 points to integrate the linewidth and thermal transport. I guess, you are just running tests, but pay attention to do proper convergence testing of both.

kind regards
